src/main/java/jpa/domain/Favorite.java

```diff
@@ -1,15 +1,40 @@
 package jpa.domain;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
+import javax.persistence.*;
 
 @Entity
 public class Favorite extends BaseTimeEntity {
-    @Id @GeneratedValue
-    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "DEPART_STATION_ID")
```

Comment:
일반적으로 데이터베이스의 컬럼 이름에는 대문자를 쓰지 않는것 같아요

```suggestion
    @JoinColumn(name = "depart_station_id")
```